[opencv4[halide]] Build failure

Package: opencv4[core,default-features,dnn,halide,jpeg,png,quirc,tiff,webp]:arm64-osx -> 4.8.0

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 576f3f1ad 2023-08-14 (13 hours ago)

**To Reproduce**

`vcpkg x-set-installed --enforce-port-checks --allow-unsupported opencv4[halide]`

**Failure logs**

```
-- Using cached opencv4-e5e1a3bfdea96bebda2ad963bc8f6cf17930aef7.patch.
-- Using cached opencv-opencv-4.8.0.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/opencv-opencv-4.8.0.tar.gz
-- Applying patch 0001-disable-downloading.patch
-- Applying patch 0002-install-options.patch
-- Applying patch 0003-force-package-requirements.patch
-- Applying patch 0004-fix-eigen.patch
-- Applying patch 0005-fix-policy-CMP0057.patch
-- Applying patch 0006-fix-uwp.patch
-- Applying patch 0008-devendor-quirc.patch
-- Applying patch 0009-fix-protobuf.patch
-- Applying patch 0010-fix-uwp-tiff-imgcodecs.patch
-- Applying patch 0011-remove-python2.patch
-- Applying patch 0012-fix-zlib.patch
-- Applying patch 0015-fix-freetype.patch
-- Applying patch /Users/leanderSchulten/git_projekte/vcpkg/downloads/opencv4-e5e1a3bfdea96bebda2ad963bc8f6cf17930aef7.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean
-- Using cached opencv-cache/tiny_dnn/adb1c512e09ca2c7a6faef36f9c53e59-v1.0.0a3.tar.gz.
-- Configuring arm64-osx
-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /opt/homebrew/Cellar/cmake/3.27.1/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/opencv4/portfile.cmake:455 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/install-arm64-osx-dbg-out.log</summary>

```
Change Dir: '/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg'

Run Build Command(s): /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/ninja/1.10.2-osx/ninja -v -v -j11 install
[1/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCAROTENE_NS=carotene_o4t -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/carotene/hal/../include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/carotene/hal -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -funwind-tables -fwrapv -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT 3rdparty/carotene/hal/CMakeFiles/tegra_hal.dir/dummy.cpp.o -MF 3rdparty/carotene/hal/CMakeFiles/tegra_hal.dir/dummy.cpp.o.d -o 3rdparty/carotene/hal/CMakeFiles/tegra_hal.dir/dummy.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/carotene/hal/dummy.cpp
[2/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCAROTENE_NS=carotene_o4t -DWITH_NEON -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/carotene/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -funwind-tables -fwrapv -Wno-unused-function -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT 3rdparty/carotene/hal/carotene/CMakeFiles/carotene_objs.dir/src/bitwise.cpp.o -MF 3rdparty/carotene/hal/carotene/CMakeFiles/carotene_objs.dir/src/bitwise.cpp.o.d -o 3rdparty/carotene/hal/carotene/CMakeFiles/carotene_objs.dir/src/bitwise.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/carotene/src/bitwise.cpp
...
Skipped 467 lines
...

    return _VSTD::min<size_type>(__alloc_traits::max_size(this->__alloc()),
                                                 ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:680:19: note: in instantiation of member function 'std::vector<std::thread>::max_size' requested here
        if (__n > max_size())
                  ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:1081:9: note: in instantiation of member function 'std::vector<std::thread>::__vallocate' requested here
        __vallocate(__n);
        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:74:30: note: in instantiation of member function 'std::vector<std::thread>::vector' requested here
    std::vector<std::thread> threads(num_threads - 1);
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__threading_support:595:24: note: forward declaration of 'std::thread'
class _LIBCPP_TYPE_VIS thread;
                       ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:8:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/precomp.hpp:49:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core.hpp:52:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/cvdef.h:788:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/array:127:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/algorithm:1712:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/memory:848:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__memory/allocator.h:110:59: error: invalid application of 'sizeof' to an incomplete type 'std::thread'
            return static_cast<_Tp*>(::operator new(__n * sizeof(_Tp)));
                                                          ^~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__memory/allocate_at_least.h:54:19: note: in instantiation of member function 'std::allocator<std::thread>::allocate' requested here
  return {__alloc.allocate(__n), __n};
                  ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:1081:9: note: in instantiation of member function 'std::vector<std::thread>::__vallocate' requested here
        __vallocate(__n);
        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:74:30: note: in instantiation of member function 'std::vector<std::thread>::vector' requested here
    std::vector<std::thread> threads(num_threads - 1);
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__threading_support:595:24: note: forward declaration of 'std::thread'
class _LIBCPP_TYPE_VIS thread;
                       ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:8:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/precomp.hpp:49:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core.hpp:53:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/base.hpp:58:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/cvstd.hpp:56:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/string:551:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/string_view:223:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/functional:515:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__functional/boyer_moore_searcher.h:26:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:754:54: error: arithmetic on a pointer to an incomplete type 'const std::vector<std::thread>::value_type' (aka 'const std::thread')
      __annotate_contiguous_container(data(), data() + capacity(),
                                              ~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:686:9: note: in instantiation of member function 'std::vector<std::thread>::__annotate_new' requested here
        __annotate_new(0);
        ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:1081:9: note: in instantiation of member function 'std::vector<std::thread>::__vallocate' requested here
        __vallocate(__n);
        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:74:30: note: in instantiation of member function 'std::vector<std::thread>::vector' requested here
    std::vector<std::thread> threads(num_threads - 1);
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__threading_support:595:24: note: forward declaration of 'std::thread'
class _LIBCPP_TYPE_VIS thread;
                       ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:8:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/precomp.hpp:49:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core.hpp:53:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/base.hpp:58:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/cvstd.hpp:56:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/string:551:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/string_view:223:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/functional:515:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__functional/boyer_moore_searcher.h:26:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:540:52: error: arithmetic on a pointer to an incomplete type 'std::thread'
        {return static_cast<size_type>(__end_cap() - this->__begin_);}
                                       ~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:754:56: note: in instantiation of member function 'std::vector<std::thread>::capacity' requested here
      __annotate_contiguous_container(data(), data() + capacity(),
                                                       ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:686:9: note: in instantiation of member function 'std::vector<std::thread>::__annotate_new' requested here
        __annotate_new(0);
        ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:1081:9: note: in instantiation of member function 'std::vector<std::thread>::__vallocate' requested here
        __vallocate(__n);
        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:74:30: note: in instantiation of member function 'std::vector<std::thread>::vector' requested here
    std::vector<std::thread> threads(num_threads - 1);
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__threading_support:595:24: note: forward declaration of 'std::thread'
class _LIBCPP_TYPE_VIS thread;
                       ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:8:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/precomp.hpp:49:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core.hpp:53:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/base.hpp:58:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/cvstd.hpp:56:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/string:551:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/string_view:223:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/functional:515:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__functional/boyer_moore_searcher.h:26:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:996:73: error: arithmetic on a pointer to an incomplete type 'std::thread'
    for (pointer __pos = __tx.__pos_; __pos != __new_end; __tx.__pos_ = ++__pos) {
                                                                        ^ ~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:1082:9: note: in instantiation of member function 'std::vector<std::thread>::__construct_at_end' requested here
        __construct_at_end(__n);
        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:74:30: note: in instantiation of member function 'std::vector<std::thread>::vector' requested here
    std::vector<std::thread> threads(num_threads - 1);
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__threading_support:595:24: note: forward declaration of 'std::thread'
class _LIBCPP_TYPE_VIS thread;
                       ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:8:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/precomp.hpp:49:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core.hpp:53:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/base.hpp:58:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/cvstd.hpp:56:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/string:551:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/string_view:223:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/functional:515:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__functional/boyer_moore_searcher.h:26:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:781:62: error: arithmetic on a pointer to an incomplete type 'std::thread'
      : __v_(__v), __pos_(__v.__end_), __new_end_(__v.__end_ + __n) {
                                                  ~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:994:27: note: in instantiation of member function 'std::vector<std::thread>::_ConstructTransaction::_ConstructTransaction' requested here
    _ConstructTransaction __tx(*this, __n);
                          ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:1082:9: note: in instantiation of member function 'std::vector<std::thread>::__construct_at_end' requested here
        __construct_at_end(__n);
        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:74:30: note: in instantiation of member function 'std::vector<std::thread>::vector' requested here
    std::vector<std::thread> threads(num_threads - 1);
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__threading_support:595:24: note: forward declaration of 'std::thread'
class _LIBCPP_TYPE_VIS thread;
                       ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:8:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/precomp.hpp:49:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core.hpp:53:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/base.hpp:58:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/cvstd.hpp:56:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/string:551:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/string_view:223:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/functional:515:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__functional/boyer_moore_searcher.h:26:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:833:64: error: arithmetic on a pointer to an incomplete type 'std::thread'
        __alloc_traits::destroy(__alloc(), _VSTD::__to_address(--__soon_to_be_end));
                                                               ^ ~~~~~~~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:827:29: note: in instantiation of member function 'std::vector<std::thread>::__base_destruct_at_end' requested here
  void __clear() _NOEXCEPT {__base_destruct_at_end(this->__begin_);}
                            ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:436:9: note: in instantiation of member function 'std::vector<std::thread>::__clear' requested here
        __clear();
        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:74:30: note: in instantiation of member function 'std::vector<std::thread>::~vector' requested here
    std::vector<std::thread> threads(num_threads - 1);
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__threading_support:595:24: note: forward declaration of 'std::thread'
class _LIBCPP_TYPE_VIS thread;
                       ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:8:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/precomp.hpp:49:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core.hpp:52:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include/opencv2/core/cvdef.h:788:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/array:127:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/algorithm:1712:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/memory:848:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__memory/allocator.h:128:58: error: invalid application of 'sizeof' to an incomplete type 'std::thread'
            _VSTD::__libcpp_deallocate((void*)__p, __n * sizeof(_Tp), _LIBCPP_ALIGNOF(_Tp));
                                                         ^~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__memory/allocator_traits.h:282:13: note: in instantiation of member function 'std::allocator<std::thread>::deallocate' requested here
        __a.deallocate(__p, __n);
            ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/vector:437:25: note: in instantiation of member function 'std::allocator_traits<std::allocator<std::thread>>::deallocate' requested here
        __alloc_traits::deallocate(__alloc(), this->__begin_, capacity());
                        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_halide.cpp:74:30: note: in instantiation of member function 'std::vector<std::thread>::~vector' requested here
    std::vector<std::thread> threads(num_threads - 1);
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/__threading_support:595:24: note: forward declaration of 'std::thread'
class _LIBCPP_TYPE_VIS thread;
                       ^
14 errors generated.
[379/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_PLUGINS -DHAVE_FLATBUFFERS=1 -DHAVE_PROTOBUF=1 -DOPENCV_DNN_EXTERNAL_PROTOBUF=1 -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/dnn -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/misc/tflite -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/flatbuffers/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated -Wno-missing-prototypes -Wno-missing-declarations -Wno-shadow -Wno-unused-parameter -Wno-sign-compare -Wno-invalid-offsetof -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/dnn/CMakeFiles/opencv_dnn.dir/src/registry.cpp.o -MF modules/dnn/CMakeFiles/opencv_dnn.dir/src/registry.cpp.o.d -o modules/dnn/CMakeFiles/opencv_dnn.dir/src/registry.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/registry.cpp
[380/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_PLUGINS -DHAVE_FLATBUFFERS=1 -DHAVE_PROTOBUF=1 -DOPENCV_DNN_EXTERNAL_PROTOBUF=1 -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/dnn -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/misc/tflite -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/flatbuffers/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated -Wno-missing-prototypes -Wno-missing-declarations -Wno-shadow -Wno-unused-parameter -Wno-sign-compare -Wno-invalid-offsetof -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/dnn/CMakeFiles/opencv_dnn.dir/src/onnx/onnx_importer.cpp.o -MF modules/dnn/CMakeFiles/opencv_dnn.dir/src/onnx/onnx_importer.cpp.o.d -o modules/dnn/CMakeFiles/opencv_dnn.dir/src/onnx/onnx_importer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/onnx/onnx_importer.cpp
[381/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_PLUGINS -DHAVE_FLATBUFFERS=1 -DHAVE_PROTOBUF=1 -DOPENCV_DNN_EXTERNAL_PROTOBUF=1 -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/dnn -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/misc/tflite -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/flatbuffers/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated -Wno-missing-prototypes -Wno-missing-declarations -Wno-shadow -Wno-unused-parameter -Wno-sign-compare -Wno-invalid-offsetof -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/dnn/CMakeFiles/opencv_dnn.dir/src/op_timvx.cpp.o -MF modules/dnn/CMakeFiles/opencv_dnn.dir/src/op_timvx.cpp.o.d -o modules/dnn/CMakeFiles/opencv_dnn.dir/src/op_timvx.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_timvx.cpp
[382/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_PLUGINS -DHAVE_FLATBUFFERS=1 -DHAVE_PROTOBUF=1 -DOPENCV_DNN_EXTERNAL_PROTOBUF=1 -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/dnn -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/misc/tflite -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/flatbuffers/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated -Wno-missing-prototypes -Wno-missing-declarations -Wno-shadow -Wno-unused-parameter -Wno-sign-compare -Wno-invalid-offsetof -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/dnn/CMakeFiles/opencv_dnn.dir/src/op_vkcom.cpp.o -MF modules/dnn/CMakeFiles/opencv_dnn.dir/src/op_vkcom.cpp.o.d -o modules/dnn/CMakeFiles/opencv_dnn.dir/src/op_vkcom.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_vkcom.cpp
[383/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_PLUGINS -DHAVE_FLATBUFFERS=1 -DHAVE_PROTOBUF=1 -DOPENCV_DNN_EXTERNAL_PROTOBUF=1 -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/dnn -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/misc/tflite -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/flatbuffers/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated -Wno-missing-prototypes -Wno-missing-declarations -Wno-shadow -Wno-unused-parameter -Wno-sign-compare -Wno-invalid-offsetof -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/dnn/CMakeFiles/opencv_dnn.dir/src/tensorflow/tf_io.cpp.o -MF modules/dnn/CMakeFiles/opencv_dnn.dir/src/tensorflow/tf_io.cpp.o.d -o modules/dnn/CMakeFiles/opencv_dnn.dir/src/tensorflow/tf_io.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/tensorflow/tf_io.cpp
[384/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_PLUGINS -DHAVE_FLATBUFFERS=1 -DHAVE_PROTOBUF=1 -DOPENCV_DNN_EXTERNAL_PROTOBUF=1 -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/dnn -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/misc/tflite -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/flatbuffers/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated -Wno-missing-prototypes -Wno-missing-declarations -Wno-shadow -Wno-unused-parameter -Wno-sign-compare -Wno-invalid-offsetof -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/dnn/CMakeFiles/opencv_dnn.dir/src/torch/THGeneral.cpp.o -MF modules/dnn/CMakeFiles/opencv_dnn.dir/src/torch/THGeneral.cpp.o.d -o modules/dnn/CMakeFiles/opencv_dnn.dir/src/torch/THGeneral.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/torch/THGeneral.cpp
[385/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_PLUGINS -DHAVE_FLATBUFFERS=1 -DHAVE_PROTOBUF=1 -DOPENCV_DNN_EXTERNAL_PROTOBUF=1 -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/dnn -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/misc/tflite -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/flatbuffers/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated -Wno-missing-prototypes -Wno-missing-declarations -Wno-shadow -Wno-unused-parameter -Wno-sign-compare -Wno-invalid-offsetof -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/dnn/CMakeFiles/opencv_dnn.dir/src/op_webnn.cpp.o -MF modules/dnn/CMakeFiles/opencv_dnn.dir/src/op_webnn.cpp.o.d -o modules/dnn/CMakeFiles/opencv_dnn.dir/src/op_webnn.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/op_webnn.cpp
[386/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_PLUGINS -DHAVE_FLATBUFFERS=1 -DHAVE_PROTOBUF=1 -DOPENCV_DNN_EXTERNAL_PROTOBUF=1 -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/dnn -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/misc/tflite -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/flatbuffers/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated -Wno-missing-prototypes -Wno-missing-declarations -Wno-shadow -Wno-unused-parameter -Wno-sign-compare -Wno-invalid-offsetof -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/dnn/CMakeFiles/opencv_dnn.dir/src/tflite/tflite_importer.cpp.o -MF modules/dnn/CMakeFiles/opencv_dnn.dir/src/tflite/tflite_importer.cpp.o.d -o modules/dnn/CMakeFiles/opencv_dnn.dir/src/tflite/tflite_importer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/tflite/tflite_importer.cpp
[387/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_PLUGINS -DHAVE_FLATBUFFERS=1 -DHAVE_PROTOBUF=1 -DOPENCV_DNN_EXTERNAL_PROTOBUF=1 -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/dnn -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/misc/tflite -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/flatbuffers/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated -Wno-missing-prototypes -Wno-missing-declarations -Wno-shadow -Wno-unused-parameter -Wno-sign-compare -Wno-invalid-offsetof -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/dnn/CMakeFiles/opencv_dnn.dir/src/tensorflow/tf_importer.cpp.o -MF modules/dnn/CMakeFiles/opencv_dnn.dir/src/tensorflow/tf_importer.cpp.o.d -o modules/dnn/CMakeFiles/opencv_dnn.dir/src/tensorflow/tf_importer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/tensorflow/tf_importer.cpp
[388/576] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_PLUGINS -DHAVE_FLATBUFFERS=1 -DHAVE_PROTOBUF=1 -DOPENCV_DNN_EXTERNAL_PROTOBUF=1 -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/dnn -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/misc/tflite -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/flatbuffers/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated -Wno-missing-prototypes -Wno-missing-declarations -Wno-shadow -Wno-unused-parameter -Wno-sign-compare -Wno-invalid-offsetof -g  -O0 -DDEBUG -D_DEBUG -std=c++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/dnn/CMakeFiles/opencv_dnn.dir/src/tensorflow/tf_graph_simplifier.cpp.o -MF modules/dnn/CMakeFiles/opencv_dnn.dir/src/tensorflow/tf_graph_simplifier.cpp.o.d -o modules/dnn/CMakeFiles/opencv_dnn.dir/src/tensorflow/tf_graph_simplifier.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/dnn/src/tensorflow/tf_graph_simplifier.cpp
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "gdal",
      "default-features": false,
      "features": [
        "poppler"
      ]
    }
  ],
  "$overrides": [
    {
      "name": "capstone",
      "version": "4.0.1-120373dc"
    }
  ],
  "$builtin-baseline": "338f99eed499b6df1628829e40695b396a70a1f7"
}

```
</details>

--- a/ports/opencv3/0019-missing-include.patch
+++ b/ports/opencv3/0019-missing-include.patch
@@ -1,0 +1,12 @@
+diff --git a/modules/dnn/src/op_halide.cpp b/modules/dnn/src/op_halide.cpp
+index c96971bc6aac..d107e01887d0 100644
+--- a/modules/dnn/src/op_halide.cpp
++++ b/modules/dnn/src/op_halide.cpp
+@@ -11,6 +11,7 @@
+ 
+ #ifdef HAVE_HALIDE
+ #include <HalideRuntimeOpenCL.h>
++#include <thread>
+ #endif  // HAVE_HALIDE
+ 
+ namespace cv

--- a/ports/opencv3/portfile.cmake
+++ b/ports/opencv3/portfile.cmake
@@ -22,6 +22,7 @@ vcpkg_from_github(
       0010-fix-uwp-tiff-imgcodecs.patch
       0011-remove-python2.patch
       0012-fix-zlib.patch
+      0019-missing-include.patch
       fix-tbb-error.patch
 )
 # Disallow accidental build of vendored copies

--- a/ports/opencv3/vcpkg.json
+++ b/ports/opencv3/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "opencv3",
   "version": "3.4.18",
-  "port-version": 10,
+  "port-version": 11,
   "description": "Open Source Computer Vision Library",
   "homepage": "https://github.com/opencv/opencv",
   "license": "BSD-3-Clause",

--- a/ports/opencv4/0019-missing-include.patch
+++ b/ports/opencv4/0019-missing-include.patch
@@ -1,0 +1,12 @@
+diff --git a/modules/dnn/src/op_halide.cpp b/modules/dnn/src/op_halide.cpp
+index 653de36146cc..db1a72278e70 100644
+--- a/modules/dnn/src/op_halide.cpp
++++ b/modules/dnn/src/op_halide.cpp
+@@ -14,6 +14,7 @@
+ #include "halide_scheduler.hpp"
+ 
+ #include <HalideRuntimeOpenCL.h>
++#include <thread>
+ #endif  // HAVE_HALIDE
+ 
+ namespace cv {

--- a/ports/opencv4/portfile.cmake
+++ b/ports/opencv4/portfile.cmake
@@ -27,6 +27,7 @@ vcpkg_from_github(
       0012-fix-zlib.patch
       0015-fix-freetype.patch
       0017-fix-flatbuffers.patch
+      0019-missing-include.patch
       "${ARM64_WINDOWS_FIX}"
 )
 # Disallow accidental build of vendored copies

--- a/ports/opencv4/vcpkg.json
+++ b/ports/opencv4/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "opencv4",
   "version": "4.8.0",
-  "port-version": 2,
+  "port-version": 3,
   "description": "computer vision library",
   "homepage": "https://github.com/opencv/opencv",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6010,11 +6010,11 @@
     },
     "opencv3": {
       "baseline": "3.4.18",
-      "port-version": 10
+      "port-version": 11
     },
     "opencv4": {
       "baseline": "4.8.0",
-      "port-version": 2
+      "port-version": 3
     },
     "opendnp3": {
       "baseline": "3.1.1",

--- a/versions/o-/opencv3.json
+++ b/versions/o-/opencv3.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "8eca838beb277535a53756ceb5da17aa8b5050a8",
+      "version": "3.4.18",
+      "port-version": 11
+    },
+    {
       "git-tree": "b66d5bf3d61ae3bd5e71952b7642658d8562ae5d",
       "version": "3.4.18",
       "port-version": 10

--- a/versions/o-/opencv4.json
+++ b/versions/o-/opencv4.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "d22421d8b63a7e2221f91520e9dcfcce1ca7bb48",
+      "version": "4.8.0",
+      "port-version": 3
+    },
+    {
       "git-tree": "89d6da3b45b7e1667bdc3083f9641e57ad4930a9",
       "version": "4.8.0",
       "port-version": 2

